parm/config/config.wave

```diff
@@ -113,8 +113,17 @@ fi
 
 # Determine if wave component needs input and/or is coupled
 export WW3ATMINP='CPL'
-export WW3ICEINP='YES'
-export WW3CURINP='YES'
+if [[ $DO_ICE == "YES" ]]; then
```

Comment:
This should be based on the app not on "DO_*" component variables.

Reply:
Would be interested in hearing why. While I'm not necessarily opposed, I also don't want that section to become overly large.

Reply:
What the wave model is coupled to or not is based on the APP, not on if "DO_COMP" variables existence.  This is also based on which nems.configure template is used and therefore would be more appropriate to be based on the apps.

Reply:
Open an issue and I'll change it. With the way the workflow is set up currently, it is functionally identical. Really it sounds like we should be determining the APP and everything from a nems.configure template name instead of the reverse.